crowbar_framework/config/initializers/delayed_job_config.rb

```diff
@@ -14,12 +14,27 @@
 # limitations under the License.
 #
 
+module Delayed
+  class Worker
+    if Rails.env.production?
```

Comment:
This is so ugly, what a crappy check. sure, 2 vars instead of one but really? The code needs to be read by actual persons, not robots. :angry: 
```
@log_file = if Rails.env.production?
                  File.join(ENV["CROWBAR_LOG_DIR"], "background_jobs.log")
                else 
                  File.join(Rails.root, "log", "background_jobs.log")
                end
```